daemon/container_operations_unix.go

```diff
@@ -71,7 +71,7 @@ func (daemon *Daemon) getIpcContainer(id string) (*container.Container, error) {
 	// Check the container ipc is shareable
 	if st, err := os.Stat(container.ShmPath); err != nil || !st.IsDir() {
 		if err == nil || os.IsNotExist(err) {
-			return nil, errors.New(errMsg + ": non-shareable IPC")
+			return nil, errors.New(errMsg + ": non-shareable IPC (hint: use --ipc shareable for the donor container)")
```

Comment:
We should avoid mentioning flags here :smile:

Reply:
done (it's "IpcMode: shareable" now)